python/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    webtest: marks tests as making a network call (deselect with '-m "not webtest"')
```

Comment:
shall we use the standard test terms , like unit / integration / system / smoke?